pkg/mutation/reporter/stats_reporter.go

```diff
@@ -0,0 +1,161 @@
+package reporter
+
+import (
+	"context"
+	"time"
+
+	"github.com/open-policy-agent/gatekeeper/pkg/metrics"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+const (
+	mutatorIngestionCountMetricName    = "mutator_ingestion_count"
+	mutatorIngestionDurationMetricName = "mutator_ingestion_duration_seconds"
+	mutatorsMetricName                 = "mutators"
+	mutationSystemIterationsMetricName = "mutation_system_iterations"
+)
+
+// MutatorIngestionStatus defines the outcomes of an attempt to add a Mutator to the mutation System.
+type MutatorIngestionStatus string
+
+// SystemConvergenceStatus defines the outcomes of the attempted mutation of an object by the
+// mutation System.  The System is meant to converge on a fully mutated object.
+type SystemConvergenceStatus string
+
+var (
+	mutatorStatusKey = tag.MustNewKey("status")
+
+	// MutatorStatusActive denotes a successfully ingested mutator, ready to mutate objects.
+	MutatorStatusActive MutatorIngestionStatus = "active"
+	// MutatorStatusError denotes a mutator that failed to ingest.
+	MutatorStatusError MutatorIngestionStatus = "error"
+
+	systemConvergenceKey = tag.MustNewKey("success")
+
+	// SystemConvergenceTrue denotes a successfully converged mutation system request.
+	SystemConvergenceTrue SystemConvergenceStatus = "true"
```

Comment:
Are we unable to just use a bool here?

Reply:
See the function signature of `tag.New()`:

```
func Insert(k Key, v string, mds ...Metadata) Mutator {
```

It expects a string, so I think it's easiest to use the custom string type like I've made above.  A boolean would just mean we need to switch it back into a string before inserting it.